CMakeLists.txt

```diff
@@ -112,7 +112,9 @@ if(SFML_OS_ANDROID)
     endif()
 
     # install everything in $NDK/sources/ because this path is appended by the NDK (convenient)
-    set(CMAKE_INSTALL_PREFIX ${CMAKE_ANDROID_NDK}/sources/third_party/sfml)
+    if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+        set(CMAKE_INSTALL_PREFIX ${CMAKE_ANDROID_NDK}/sources/third_party/sfml CACHE PATH "..." FORCE)
```

Comment:
You should add a description instead of `...`, which will be displayed when hovering over the variable.

Since we require CMake 3.7.x for Android and the variable only exists since CMake 3.7.x, this change seems okay otherwise.

Reply:
Oops, sorry I've added it verbatim. I change this immediately.

Reply:
Done.